imgee/__init__.py

```diff
@@ -46,6 +47,3 @@ def error403(error):
         app.config['MEDIA_DOMAIN'].startswith('http:') or
         app.config['MEDIA_DOMAIN'].startswith('https:')):
```

Comment:
Replace this if condition with `if app.config.get('MEDIA_DOMAIN', '').lower().startswith(('http://', 'https://'))`